docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,112 @@
+# Setting up a microk8s instance
+
+[Microk8s](https://microk8s.io/) is a single-package fully conformant lightweight Kubernetes that works on 42 Linux versions. The project is publicly available and can be [found here](https://github.com/ubuntu/microk8s).
+
+Follow the steps below to set up a microk8s instance from scratch, including all the necessary dependencies and configuration.
+
+## 1. Concepts
+
+### Helm and tiller
+
+Two executables will get installed onto the cluster as part of this process: `helm` and `tiller`.
+
+-   `helm` is responsible for resolving the configuration of the chart to be installed, while issuing the correct install commands onto the cluster.
+-   `tiller` is responsible for receiving the install commands issued by `helm`, as well as managing the lifecycle of the components that have been installed.
+
+`helm` is the client facing side, while `tiller` is the server/cluster facing side.
+
+## 2. Prepare your environment
+
+Prepare your environment to set up the microk8s cluster.
+For your infrastructure, you will need the following:
+
+1.  A machine running Ubuntu 18.04 LTS.
+2.  A postgres database with all the necessary databases created and open firewall rule. ([Documentation can be found here.](../requirements.md#postgresql-server-setup))
+3.  Establish a session (local or remote) onto the machine described in 1. above.
+
+All the following steps assume that you are starting from a blank slate.
+
+## 3. Installing microk8s
+
+1.  Make sure the machine is up to date and also has the `nfs-common` package installed
+
+    ```bash
+    sudo apt update && sudo apt install nfs-common -y
+    ```
+
+2.  Install the `1.15/stable` channel of microk8s.
+
+    ```bash
+    sudo snap install microk8s --classic --channel=1.15/stable && \
+    sudo usermod -a -G microk8s $USER  && \
+    su - $USER
+    ```
+
+    Check if everything was installed correctly.
+
+    ```bash
+    microk8s.status --wait-ready
+    ```
+
+3.  Install the helm binary
+
+    ```bash
+    HELM_PKG=helm-v2.16.3-linux-amd64.tar.gz
+    wget https://get.helm.sh/$HELM_PKG
+    tar xvzf $HELM_PKG
+    mv linux-amd64/tiller linux-amd64/helm /usr/local/bin
```

Comment:
confirm if this needs sudo.. according to andrea it does

Reply:
It does, so that you can write on the folder /usr/local/bin.